sdk/python/jobs/automl-standalone-jobs/automl-forecasting-github-dau/auto-ml-forecasting-github-dau.ipynb

```diff
@@ -0,0 +1,841 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.jobs.stream(returned_job.name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 4. Retrieve the Best Trial (Best Model's trial/run)\n",
+    "Use the MLFLowClient to access the results (such as Models, Artifacts, Metrics) of a previously completed AutoML Trial."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.1 Initialize MLFlow Client\n",
+    "The models and artifacts that are produced by AutoML can be accessed via the MLFlow interface. \n",
+    "Initialize the MLFlow client here, and set the backend as Azure ML, via. the MLFlow Client.\n",
+    "\n",
+    "*IMPORTANT*, you need to have installed the latest MLFlow packages with:\n",
+    "\n",
+    "    pip install azureml-mlflow\n",
+    "\n",
+    "    pip install mlflow"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Obtain the tracking URI for MLFlow"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import mlflow\n",
+    "\n",
+    "# Obtain the tracking URL from MLClient\n",
+    "MLFLOW_TRACKING_URI = ml_client.workspaces.get(\n",
+    "    name=ml_client.workspace_name\n",
+    ").mlflow_tracking_uri\n",
+    "\n",
+    "print(MLFLOW_TRACKING_URI)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set the MLFLOW TRACKING URI\n",
+    "\n",
+    "mlflow.set_tracking_uri(MLFLOW_TRACKING_URI)\n",
+    "\n",
+    "print(\"\\nCurrent tracking uri: {}\".format(mlflow.get_tracking_uri()))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from mlflow.tracking.client import MlflowClient\n",
+    "\n",
+    "# Initialize MLFlow client\n",
+    "mlflow_client = MlflowClient()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get the AutoML parent Job"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job_name = returned_job.name\n",
+    "\n",
+    "# Example if providing an specific Job name/ID\n",
+    "# job_name = \"funny_soursop_2zpkp35pdy\"\n",
+    "\n",
+    "# Get the parent run\n",
+    "mlflow_parent_run = mlflow_client.get_run(job_name)\n",
+    "\n",
+    "print(\"Parent Run: \")\n",
+    "print(mlflow_parent_run)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Print parent run tags. 'automl_best_child_run_id' tag should be there.\n",
+    "print(mlflow_parent_run.data.tags)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.2 Get the AutoML best child run"
```

Comment:
1. Please remove the number from the subsection header.
2. We are not offering the same functionality as the v1 version of this notebook. If want to proceed with the route of showing users how to inference/evaluate a single (in this case, best) child run, we need to give users the ability to download validation set metrics for all HD runs and show them how to inference any of them.